tests/shared.sh

```diff
@@ -77,10 +61,36 @@ function wait_for_server_started {
       exit 1
     fi
     printf '.'
-    SLEEP_SECS=2
-    sleep $SLEEP_SECS
+    sleep 2
   done
-  printf '\nserver started\n'
+  echo "Server started"
   rm "$log"
-  sleep 4
+  sleep $sleep_time
 }
+
+function init_myvarsolr {
```

Comment:
as this is now generalised, it should be renamed `prepare_dir_to_mount` or some such.

Reply:
Done